bibliopixel/animation/animation.py

```diff
@@ -66,6 +67,8 @@ def cleanup(self, clean_layout=True):
     def compute_state(self):
         if self.threading.stop_event.isSet():
             self.state = STATE.canceled
+        elif self.runner.seconds and (time.time() - self.runner.run_start_time) > self.runner.seconds:
+            self.state = STATE.timeout
```

Comment:
Because I told it to run for n seconds and it timed out. I don't care what it's called. Not actually used by anything at the moment.